Library/Formula/xmake.rb

```diff
@@ -0,0 +1,15 @@
+class Xmake < Formula
+  desc "The Automatic Cross-platform Build Tool"
+  homepage "https://github.com/waruqi/xmake"
+  url "https://github.com/waruqi/xmake/archive/v1.0.4.tar.gz"
+  mirror "http://tboox.net/release/xmake/xmake-1.0.4.tar.gz"
+  sha256 "17527514245803a1fc46a6261fd73086022482e19645601c3d187cda96331321"
+
+  def install
+    system "./install"
+  end
+
+  test do
+    system "xmake", "--help"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of a good test. Thanks!